docker/docker-prepare.sh

```diff
@@ -6,14 +6,11 @@ wait_for_postgres() {
 
 	echo "Waiting for PostgreSQL to start..."
 
-	host="${PAPERLESS_DBHOST}"
-	port="${PAPERLESS_DBPORT}"
+	host="${PAPERLESS_DBHOST:=localhost}"
+	port="${PAPERLESS_DBPORT:=5342}"
 
-	if [[ -z $port ]]; then
-		port="5432"
-	fi
 
-	while ! </dev/tcp/$host/$port; do
+	while [ ! "$(pg_isready -h "$host" -p "$port")" ]; do
```

Comment:
No, 
i did some investigating to find a better and more clear solution. 

Regards

Schnuffle

Reply:
In docker/docker-prepare.sh line 16:
while ! </dev/tcp/$host/$port; do
 ______^-------------------^ SC2188 (warning): This redirection doesn't have a command. Move to its command (or use 'true' as no-op).
____________________^---^ SC2086 (info): Double quote to prevent globbing and word splitting.

Did you mean: 
	while ! </dev/tcp/"$host"/$port; do

Reply:
I removed the nested quotes